### ISSUE TITLE
[Feature request] Hide video buffer

### ISSUE THREAD

**Issue by 77-77SP** [abdallahmehiz/mpvKt#16]
First of all, thank you for developing this amazing app!

I would like to suggest video buffer customization option that would allow hiding the buffer indicator on the progress bar when playing local video files and only show it when  playing from URL.

**Comment by Pentaphon** [comment]
I don't think we need to hide it for local files but if it has to be done, no settings option is needed. Just hide it for local files and show it for streaming URL files.
